### DISCUSSION TITLE
image and description on user query share

### DISCUSSION THREAD

**Discussion by TomW1605** [FreshRSS/FreshRSS#6537] [Ideas]
hi
is there a way to add a description and feed image to the rss feed from a user query?

i am using FreshRSS to process the patreon feeds of some podcasts to split out normal and patreon episodes so they are different feeds in my podcast app. unfortunatly they dont have the image or description of the input feed and that makes them quite hard to tell apart.

i understand this is not at all the intended usecase for FreshRSS but other than this issue it is something it works extreemly well for so if there was a way to do this it would be basicaly perfect.
thanks

**Comment by Alkarex** [comment]
We could add a field to provide a description to be included in the RSS output feeds.
Maybe even a field to provide the URL of an image of an RSS output feed (image management would be a larger work).

But I am not sure I got your need right. Could you maybe post a screenshot, to be sure we are talking about the same thing?

**Reply by TomW1605** [reply]
basicaly, i put the patreon feed url into FreshRSS and used user queries to split out just the patreon episodes from the normal ones that are included in the same feed. i then used the sharable RSS link to add that feed to Overcast. however, because it dosnt have an image it just shows a stock placeholder in Overcast making it hard to tell them apart. attached are screenshots of what i see in Overcast and the user query.

a field for image url should be fine since that is all thats in the existing feed. another option might be to have an option to copy the feed details from an existing feed (basicaly everything in the channel tag other than items) with maybe options to override common things like title. it would be good if this can be copied from a feed other than the one the new feed is built from, that way you could have the standard episodes extracted from the patreon feed look like the public feed (for ones that also have things like ad free versions of the public episodes in the patreon feed).

![image](https://github.com/FreshRSS/FreshRSS/assets/17092573/653c2641-8c80-4673-8876-c1cfc3839b9d)
![image](https://github.com/FreshRSS/FreshRSS/assets/17092573/efa2f0cf-fb46-440d-bc76-7ff1e1e564e0)

**Reply by Alkarex** [reply]
https://github.com/FreshRSS/FreshRSS/issues/6538

**Comment by TomW1605** [comment]
added in #6541

**Comment by TomW1605** [comment]
so i just noticed, some of the other fields (eg, author) are used by podcast players. im not keen on just adding a bunch of fields to the user queries just for podcasts so are there any other practical ways to add more of these without cluttering up the ui for people who dont need them?

**Reply by Alkarex** [reply]
Are those some fields, which are static for the generated RSS feed?

**Reply by TomW1605** [reply]
yes
